dpxdt/tools/site_diff.py

```diff
@@ -228,6 +228,12 @@ def run(self,
         good_urls = set()
 
         yield heartbeat('Scanning for content')
+        
+        http_username=None
+        http_password=None
+        if FLAGS.http_username and FLAGS.http_password:
```

Comment:
Alternatively, you can remove this if statement entirely. We may want to support username with blank passwords? So perhaps the right approach is just:

```
http_username = FLAGS.http_username
http_password = FLAGS.http_password
```

And let the values default to `None`